### PR TITLE
DSD-767: Fixing Tabs bugs

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-767](https://jira.nypl.org/browse/DSD-767)

## This PR does the following:

- Passes down `id` in the `Tabs` component to its DS children component.
- Adds more tests for the `Tabs` component including a snapshot test.

## How has this been tested?

Locally and in Storybook.

## Accessibility concerns or updates

- N/A, ids for aria labels are still there but they are just more consistent.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr859-gv0xp3nsdamvxvpe3a7uprdahtdsnoab.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/620197b2046a4b8d29abca0d

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -17,6 +17,12 @@ import Button from "../Button/Button";
 import useCarouselStyles from "../../hooks/useCarouselStyles";
 import useWindowSize from "../../hooks/useWindowSize";
 
+// Internal interface used for renders `Tabs` tab and panel
```
Comment (bigfishdesign13) [inline src/components/Tabs/Tabs.tsx]: Typo: ...used for `rendering`...

**Review by bigfishdesign13 — APPROVED** [review]
One small typo, but otherwise looks good.  Also, make sure to bump up the `Latest Version` number on the story page.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr859-gv0xp3nsdamvxvpe3a7uprdahtdsnoab.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/620197b2046a4b8d29abca0d

**Pull request merged**